### PR TITLE
feat(desktop): implement previous selection audio and camera

### PR DESCRIPTION
**Pull request by r17x**
## Changes

* Change API of  `createOptionsQuery`
  * add `setOptions`: is same with `commands.setRecordOptions`.
* Change `commands.setRecordOptions` to --> `setOptions`
  
### Screen Record

https://github.com/user-attachments/assets/3fe78ae4-3576-493c-b42a-1aa90f7b5704

 
/claim #123

### CODE REVIEW

File: apps/desktop/src/routes/(window-chrome)/index.tsx [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { createEventListener } from "@solid-primitives/event-listener";
 import { cache, createAsync, redirect, useNavigate } from "@solidjs/router";
 import { createMutation, createQuery } from "@tanstack/solid-query";
 import { getVersion } from "@tauri-apps/api/app";
-import { Window } from "@tauri-apps/api/window";
```
Comment (r17x) [inline apps/desktop/src/routes/(window-chrome)/index.tsx:7]: @richiemcilroy Removed unused

File: apps/desktop/src/utils/queries.ts [hunk 2]
```diff
@@ -64,15 +64,45 @@ export const getPermissions = queryOptions({
   refetchInterval: 1000,
 });
 
+
+type PartialRecordingOptions = Omit<RecordingOptions, "captureTarget">;
 export function createOptionsQuery() {
-  const options = createQuery(() => getOptions);
+const KEY = "recordingOptionsQuery" 
+const localState = localStorage.getItem(KEY);
```
Comment (r17x) [inline apps/desktop/src/utils/queries.ts]: I'm using `localStorage` for save value `camera` and `audio`.

**Comment by richiemcilroy** [comment]
looks great!

**Pull request merged**